custom/ewsghana/templates/ewsghana/stock_transaction.html

```diff
@@ -0,0 +1,122 @@
+{% extends "reports/standard/base_template.html" %}
+{% load hq_shared_tags %}
+{% load i18n %}
+
+{% block reports-js-inline %}
+    <script>
+        var standardHQReport = new HQReport({
```

Comment:
sorry i know this approach has been taken in other reports but can you add comments to these functions that specify why they are copied in here instead of being referenced from our library files?